py/visdom/server.py

```diff
@@ -319,6 +322,36 @@ def load_state(self):
 
         return state
 
+    def load_user_settings(self):
+        settings = {}
+
+        """Determines & uses the platform-specific root directory for user configurations."""
+        if platform.system() == "Windows":
+            base_dir = os.getenv('APPDATA')
+        elif platform.system() == "Darwin": # osx
+            base_dir = os.path.expanduser('~/Library/Preferences')
+        else:
+            base_dir = os.getenv('XDG_CONFIG_HOME', os.path.expanduser('~/.config'))
```

Comment:
I'm not familiar with `XDG_CONFIG_HOME`, is this a standard directory?

Reply:
Yes, AFAIK this comes from X-Applications initially, but many CLI-tools (e.g. `git` `curl`, `vim`, ...) use the environment variable as well. (See [here](https://wiki.archlinux.org/title/XDG_Base_Directory#Support) for a more complete list.)

Reply:
Thanks for clarifying! I should probably be using this for my other CLI tools as well then 😅